main/reference/zoe-api/zoe-data-types.md

```diff
@@ -63,9 +35,8 @@ You can get information about the contract instance via these methods:
 
 ## Invitation
 
-An **Invitation** is a kind of **[Payment](/reference/ertp-api/payment.md)**. It's a non-fungible eright
-that can be held in **Payments** or **[Purses](/reference/ertp-api/purse.md)**, just like any other
-eright. An **Invitation** **Payment** is a **Payment** holding an **Invitation**.
+An **Invitation** is a **[Payment](/reference/ertp-api/payment.md)** holding a non-fungible eright created by an **[InvitationIssuer](#invitationissuer)**.
```

Comment:
There's only one InvitationIssuer per Zoe, so in the usual case it's an eright created by **the** invitationIssuer. Do you mean to hint that it's conceivable that there can be more than one invitationIssuer? I think that's not worth it.